checkov/common/util/http_utils.py

```diff
@@ -155,7 +160,8 @@ async def aiohttp_client_session_wrapper(
     # 2. ClientOSError
     async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(resolver=aiohttp.AsyncResolver())) as session:
         for i in range(request_max_tries):
-            logging.info(f"[http_utils](aiohttp_client_session_wrapper) reporting attempt {i + 1} out of {request_max_tries}")
+            logging.info(
+                f"[http_utils](aiohttp_client_session_wrapper) reporting attempt {i + 1} out of {request_max_tries}")
```

Comment:
Perhaps add it under debug mode?

Reply:
i just changed the line, i didn't add it